demos/lane_detection_demo/python/lane_detection_demo.py

```diff
@@ -0,0 +1,330 @@
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
+    args.add_argument("-m", "--model", help="Required. Path to an .xml file with a trained model.",
+                      required=True, type=str)
+    args.add_argument("-i", "--input", help="Required. Path to an image/video file. (Specify 'cam' to work with "
+                                            "camera)", required=True, type=str)
+    args.add_argument("-l", "--cpu_extension",
+                      help="Optional. Required for CPU custom layers. Absolute path to a shared library with "
+                           "the kernels implementations.", type=str, default=None)
+    args.add_argument("-d", "--device",
+                      help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD is"
+                           " acceptable. The sample will look for a suitable plugin for device specified. "
+                           "Default value is CPU", default="CPU", type=str)
+    args.add_argument("--labels", help="Optional. Labels mapping file", default=None, type=str)
+
+    return parser
+
+def softmax(X, theta = 1.0, axis = None):
+    """
+    Compute the softmax of each element along an axis of X.
+
+    Parameters
+    ----------
+    X: ND-Array. Probably should be floats.
+    theta (optional): float parameter, used as a multiplier
+        prior to exponentiation. Default = 1.0
+    axis (optional): axis to compute values along. Default is the
+        first non-singleton axis.
+
+    Returns an array the same size as X. The result will sum to 1
+    along the specified axis.
+    ----------
+    https://nolanbconaway.github.io/blog/2017/softmax-numpy.html
+    """
+
+    # make X at least 2d
+    y = np.atleast_2d(X)   #将输入的数组转化为至少两维  x y无变化
+
+    # find axis
+    if axis is None:
+        axis = next(j[0] for j in enumerate(y.shape) if j[1] > 1)  #axis=0
+
+    # multiply y against the theta parameter,
+    y = y * float(theta)
+
+    # subtract the max for numerical stability
+    y = y - np.expand_dims(np.max(y, axis = axis), axis)
+
+    # exponentiate y
+    y = np.exp(y)
+
+    # take the sum along the specified axis
+    ax_sum = np.expand_dims(np.sum(y, axis = axis), axis)
+
+    # finally: divide elementwise
+    p = y / ax_sum
+
+    # flatten if X was 1D
+    if len(X.shape) == 1: p = p.flatten()
+
+    return p
+
+def _thresh_coord(coord):
+    pts_x = coord[:, 0]   
+    mean_x = np.mean(pts_x)
+    idx = np.where(np.abs(pts_x - mean_x) < mean_x)  
+    return coord[idx[0]]
+
+def get_lane_mask(instance_seg):
+    
+    mask_list = []
+    
+    for i in range(selected_lines):
+        f = instance_seg[:, :, i] 
+        mask_img = []
+        mask_img = cv2.cvtColor(np.asarray(pil.new(mode = 'RGBA', size = (f.shape[1], f.shape[0]))), cv2.COLOR_RGBA2RGB)
```

Comment:
It is necessary to use 2 image libraries in the same time? Is it impossible create image using opencv?